python/ray/tune/schedulers/pbt.py

```diff
@@ -109,7 +109,7 @@ class PopulationBasedTraining(FIFOScheduler):
     This Tune PBT implementation considers all trials added as part of the
     PBT population. If the number of trials exceeds the cluster capacity,
     they will be time-multiplexed as to balance training progress across the
-    population.
+    population. To run multiple trials, use `tune.run(num_samples=<int>)`.
 
```

Comment:
Can we raise an error if there is only 1 member too?

Reply:
Hm, a bit hard because trials can be added later.